### DISCUSSION TITLE
How to play 8 hour audio along with region control and timeline

### DISCUSSION THREAD

**Discussion by azizgoyoyo** [katspaugh/wavesurfer.js#3568] [Q&A]
I have to play eight hour long audio along with region and timeline please help here ,also is it possible to do not load full audio at once

![2w](https://github.com/katspaugh/wavesurfer.js/assets/159013624/8f83005d-fe60-4075-b4f0-91e942589e17)

**Comment by katspaugh** [comment]
Please check the [FAQ](https://github.com/katspaugh/wavesurfer.js?tab=readme-ov-file#faq) in the readme.

**Comment by azizgoyoyo** [comment]
May be you could please provide me some sort of clue like pre decoding the
would help

**Reply by katspaugh** [reply]
Please read the FAQ first, you clearly didn't. I'm happy to answer concrete questions once you have at least tried something.

**Comment by azizgoyoyo** [comment]
Yes, I did.
